src/poset/inmem_store.go

```diff
@@ -96,6 +111,37 @@ func (s *InmemStore) Participants() (*peers.Peers, error) {
 	return s.participants, nil
 }
 
+func (s *InmemStore) setPeers(round int64, participants *peers.Peers) error {
+	//Extend PartipantEventsCache and Roots with new peers
+	//for id, peer := range participants.ById {
+	for _, peer := range participants.ById {
+		//if _, ok := s.participantEventsCache.participants.ById[id]; !ok {
```

Comment:
What is it for?

Reply:
This is the new way to set the peers in the store. This is also in preparation for dynamic peers, moving away from "OnNewPeer(func(peer *peers.Peer)" in the NewInmemStore constructor.

Reply:
Please remove this code and uncomment it.